docs/disconnected-ZTP-flow-hub-deployment.md

```diff
@@ -429,8 +429,6 @@ spec:
   clusterLabels:
     cloud: auto-detect
     vendor: auto-detect
-  workManager:
```

Comment:
(pls consider) even though OCP doesn't complain when this CR is applied, this field is not included anymore within its CRD [here](https://github.com/open-cluster-management/klusterlet-addon-controller/blob/fix-version/deploy/agent.open-cluster-management.io_klusterletaddonconfigs_crd.yaml#L34-L113)